static/routes.config.php

```diff
@@ -139,11 +139,12 @@
 		'/{group:\d+}/add/{contact:\d+}'    => [Module\Group::class, [R::GET, R::POST]],
 		'/{group:\d+}/remove/{contact:\d+}' => [Module\Group::class, [R::GET, R::POST]],
 	],
-	'/hashtag'            => [Module\Hashtag::class,      [R::GET]],
-	'/home'               => [Module\Home::class,         [R::GET]],
-	'/help[/{doc:.+}]'    => [Module\Help::class,         [R::GET]],
-	'/inbox[/{nickname}]' => [Module\Inbox::class,        [R::GET, R::POST]],
-	'/invite'             => [Module\Invite::class,       [R::GET, R::POST]],
+	'/hashtag'                  => [Module\Hashtag::class,   [R::GET]],
+	'/help[/{doc:.+}]'          => [Module\Help::class,      [R::GET]],
+	'/home'                     => [Module\Home::class,      [R::GET]],
+	'/hcard/{profile}[/{view}]' => [Module\HoverCard::class, [R::GET]],
```

Comment:
```suggestion
	'/hcard/{profile}[/{action}]' => [Module\HoverCard::class, [R::GET]],
```
Then you can use `$parameters['action']` instead of `$args->get(2)`